portfolio.css

```diff
@@ -1,1025 +1,1825 @@
+
+  .skills-28 {
+    width: 77px;
+    padding: 12px;
+    list-style: none;
+    height: 16px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #36b37e;
+    background: #fff;
+    border-radius: 8px;
+    order: 0;
+  }
+
+  .contact-form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 58px;
+    width: 375px;
+    height: 879px;
+  }
+
+  .form-text {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    padding: 0;
+    width: 327px;
+    height: 363px;
+    align-items: center;
+  }
+
+  .text-section {
+    width: 327px;
+    height: 363px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    text-align: center;
+    color: #172b4d;
+  }
+
+  form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-top: 58px;
+    gap: 24px;
+    width: 327px;
+    height: 330px;
+  }
+
+  input[type=text] {
+    height: 48px;
+    width: 327px;
+    border: 1px solid #d0d9d4;
+    border-radius: 4px;
+  }
+
+  input[type=email] {
+    height: 48px;
+    width: 327px;
+    border: 1px solid #d0d9d4;
+    border-radius: 4px;
+  }
+
+  #message {
+    height: 148px;
+    width: 327px;
+    border: 1px solid #60c095;
+    border-radius: 4px;
+  }
+
+  #name {
+    height: 48px;
+    width: 327px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    align-items: center;
+    color: #6b778c;
+  }
+
+  .contact-bg-image {
+    width: 92.81px;
+    height: 132.37px;
+    position: absolute;
+    background: url('./images/mobile_version/frames/Frame-contact.png') no-repeat;
+    left: 315px;
+    top: 6530px;
+  }
+
+  .social-media-icon {
+    display: flex;
+    flex-direction: row;
+    justify-content: baseline;
+    align-items: center;
+    height: 20px;
+    width: 170.17px;
+    padding-top: 40px;
+    padding-left: 0;
+    margin: 0;
+    gap: 18px;
+  }
+
+  .social-media-icon > li {
+    list-style: none;
+  }
+
+  .supporting-text-2 {
+    width: 375px;
+    height: 189px;
+    margin: 0;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    color: #42526e;
+    padding-top: 60px;
+  }
+
+  .contact-form button[type=submit]:hover {
+    background-color: green;
+  }
+
+  .header {
+    display: none;
+  }
+
+  .desktop-headline {
+    display: none;
+  }
+
+  .desktop-work {
+    display: none;
+  }
+
+  .desktop-about-me {
+    display: none;
+  }
+
+  .desktop-form {
+    display: none;
+  }
+
+  footer {
+    display: none;
+  }
+}
+
+/* Media queries for the desktop version breakthrough */
+
+@media only screen and (min-width: 768px) and (max-height: 3846px) {
```

Comment:
- [ ] Please kindly fix the mobile responsiveness as mentioned by the previous reviewer [Check Figmal Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)
**Your Design**
![checkReview](https://user-images.githubusercontent.com/55596970/188272815-c498d677-75c2-46cf-a8ce-cf81b3a5a9bb.png)

**Figmal Design**
![image](https://user-images.githubusercontent.com/55596970/188272881-e1cb1b53-d254-4994-a800-e10f987bb329.png)